frontend/src/app/spot/components/text-field/stories/TextField.stories.mdx

```diff
@@ -46,33 +52,83 @@ export const Template = (args) => ({
   },
 })
 
+
+The text field component affords a few optional elements:
+
+An optional icon can be added to the left edge to provide additional context when this is necessary (in a search field, for example):
+
+> Example:  A search bar with a icon to the left
+
+An icon can also optionally be placed on the right edge. This arrow can provide additional hints on how the text input field will behave; for example, a right arrow might suggest that the value will be immediately submitted, or a down arrow might indicate that the text field will open a drop modal underneath it (a list of assignees, for example).
+
+The right icon can also be converted to an icon-only link that allows the user to clear the field.
+
+> Example: A text field with a search icon, preset text and a clear icon on the right
+
+## Behaviour
+
+Clicking on a text field will move it to the focused state. 
+
+A text field can have placeholder text to provide additionanl context or information; this text is replaced by actual text that the user inputs, and is shown again only if the field is cleared.
```

Comment:
Nope, we've got a "Focus state" story under Styles. And we don't specify the colour there either, because it might eventually depend on theming. Although if we want, we can add it to that.